src/js/components/Grid/Grid.js

```diff
@@ -37,10 +37,4 @@ const Grid = forwardRef((props, ref) => {
 Grid.displayName = 'Grid';
 Grid.propTypes = GridPropTypes;
 
-Grid.available =
```

Comment:
think we just need to fix eslint changes otherwise looks good

Reply:
I think this will break existing code that is calling Grid.available. Instead of returning `true` it will now return `undefined`. Instead, I think we should just hard code this to `true` for now.

Reply:
Aren't we removing all the places we are using `grid.available` so it wouldn't matter if it is undefined?

Reply:
I thought that at some point we had advised callers to check `Grid.available` to decide whether they could rely on Grid or would need to create a fallback. I know grommet-designer used this.

Reply:
Ah okay thanks for the explanation

Reply:
Would be worth adding a comment to explain why in the code.